app/assets/javascripts/discourse/controllers/edit-category.js.es6

```diff
@@ -101,6 +101,19 @@ export default Ember.Controller.extend(ModalFunctionality, {
           self.set('deleting', false);
         }
       });
+    },
+
+    showTooltip() {
```

Comment:
I think a much cleaner way of doing this would be to bind the classname to .cannot_delete_reason, then the action can simply toggle the class name, perhaps something along these lines

```
class="{{if deleteHidden "hidden" ""}}"
```

As a rule we prefer only to reach in to $ if we have no other option